skills/deepnote/references/cli-sync.md

```
@@ -0,0 +1,75 @@
+deepnote sync workspace --dry-run
+
+# Machine-readable summary
+deepnote sync workspace -o json
+```
+
+## How sync decides
+
+State lives in `.deepnote-sync.json` in the synced directory: a map of project id → local path,
+last-synced `metadata.modifiedAt`, and a content hash. Projects are tracked by id because names
+(projects and folders) are **not unique** in Deepnote — cloud renames become local file moves, and
+path collisions get a deterministic ` (<short id>)` suffix.
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)